src/sidebar/panel.js

```diff
@@ -55,12 +55,12 @@ function handleLocalContent(data) {
   if (!data.hasOwnProperty('notes')) {
     quill.setContents({
       ops: [
-        { attributes: { size: 'large', bold: true }, insert: browser.i18n.getMessage('welcomeTitle') },
+        { attributes: { size: 'large', bold: true }, insert: browser.i18n.getMessage('welcomeTitle2') },
         { insert: '\n\n', attributes: { direction: LANG_DIR, align: TEXT_ALIGN_DIR }},
         {
           attributes: { size: 'large' },
           insert:
-            browser.i18n.getMessage('welcomeText')
+            browser.i18n.getMessage('welcomeText2')
```

Comment:
When we change ids like that, shouldn't we fallback on the previous id value? Especially for languages that haven't the new string translation yet?